src/test/resources/data/FeedbackParticipantFeedbackResponseCommentDeleteTest.json

```diff
@@ -270,6 +270,28 @@
       "showRecipientNameTo": [
         "RECEIVER"
       ]
+    },
```

Comment:
Actually I tried adding tests in `FeedbackParticipantFeedbackResponseCommentDeleteTest` same as MCQ, but it was becoming unnecessarily long. And I also thought that this action only tests deletion of feedback participant comment entity, so adding for all types of questions may not be needed. What do you think?

Reply:
I agree with you that adding repetitive tests for questions is not necessary since they have the same logic. However, is it possible for you to change some of your existing tests to be MSQs? That means you cover all possible logics, and also all the questions, but not necessary all logics for all questions.